staging/src/k8s.io/cloud-provider/controllers/service/metrics.go

```diff
@@ -32,12 +33,26 @@ var register sync.Once
 // registerMetrics registers service-controller metrics.
 func registerMetrics() {
 	register.Do(func() {
+		legacyregistry.MustRegister(loadBalancerSyncCount)
 		legacyregistry.MustRegister(nodeSyncLatency)
+		legacyregistry.MustRegister(nodeSyncErrorCount)
 		legacyregistry.MustRegister(updateLoadBalancerHostLatency)
 	})
 }
 
 var (
+	loadBalancerSyncCount = metrics.NewCounter(&metrics.CounterOpts{
+		Name:           "load_balancer_sync_total",
```

Comment:
super nit:  below we have "loadbalancer" and here we have "load_balancer".  Since the "update_loadbalancer_host_latency_seconds" let's go with "loadbalancer" ?

I admit to being kind of annoyed at "nodesync" vs "loadbalancer_sync", but oh well :)